buildall.sh

```diff
@@ -1,18 +1,22 @@
-mvn install
-mvn install -Dsecure=all
-mvn install -Dsecure=ui
-mvn install -Ddb=mysql
-mvn install -Ddb=mysql -Dsecure=all
-mvn install -Ddb=mysql -Dsecure=ui
-mvn install -Ddb=mariadb
-mvn install -Ddb=mariadb -Dsecure=all
-mvn install -Ddb=mariadb -Dsecure=ui
-mvn install -Ddb=oracle
-mvn install -Ddb=oracle -Dsecure=all
-mvn install -Ddb=oracle -Dsecure=ui
-mvn install -Ddb=sqlserver
-mvn install -Ddb=sqlserver -Dsecure=all
-mvn install -Ddb=sqlserver -Dsecure=ui
-mvn install -Ddb=db2
-mvn install -Ddb=firebird
-mvn install -Ddb=h2
+#!/bin/bash
+
+set -e
```

Comment:
This ensures the whole script aborts and fails in case any of the below build commands fail.